bin/release-notification/main.rb

```diff
@@ -132,7 +132,7 @@ def formated_git_commits_count
   # Display message from fixes tag.
   class FixesLog
 
-    FIXES_LINE_REGEX = Regexp.new('^ *(?:Fixes|Closes) *:? *(?<line>.*)')
+    FIXES_LINE_REGEX = Regexp.new('^ *(?:Fixes|Closes) *:? *(?<line>.*)', true)
```

Comment:
If this is case insensitive, remove capital letters
```suggestion
    FIXES_LINE_REGEX = Regexp.new('^ *(?:fixes|closes) *:? *(?<line>.*)', true)
```